extensions/integration-tests/src/notebook.test.ts

```diff
@@ -104,9 +104,49 @@ if (context.RunTest) {
 				assert(sparkResult === '2', `Expected spark result: 2, Actual: ${sparkResult}`);
 			});
 		}
+
+		test('scala language test', async function () {
+			let language = 'scala';
+
+			languageTest(pySparkNotebookContent, pySpark3KernelMetadata, this.test.title, language, {
+				name: language,
+				version: '',
+				mimetype: ''
+			});
+		});
+
+		test('cplusplus language test', async function () {
+			let language = 'cplusplus';
+			languageTest(pySparkNotebookContent, pySpark3KernelMetadata, this.test.title, language, {
+				name: language,
+				version: '',
+				mimetype: ''
+			});
+		});
+
+		test('Empty language test', async function () {
+			let language = '';
+			languageTest(pySparkNotebookContent, pySpark3KernelMetadata, this.test.title, language, {
```

Comment:
These calls need to be awaited to run properly - right now they'll never fail because they immediately return.